### PR TITLE
Graph comparison

### PR DESCRIPTION
**Pull request by GbotemiB**
This is a simple script that returns the graph properties of a pypsa-network

### CODE REVIEW

File: compare_graph.py
Comment (jessLryan) [inline compare_graph.py]: the above code looks great!
Let's leave the code below out as its purpose is to experiment with the above code - for now let's keep experimentation local, we can set up some separate experimentation scripts at a later date

Comment (GbotemiB) [reply compare_graph.py]: I think we can still leave the experimentation, as it only runs when you run the script alone. If you import the script as a module, the experimentation wont run.

Comment (GbotemiB) [reply compare_graph.py]: Depending on what we later agree on. I will just comment it out for now.

Comment (jessLryan) [reply compare_graph.py]: the repo is for doing a general network comparison so shouldn't involve any specific files. It's also not good practice to have commented out code in a repository

Comment (jessLryan) [reply compare_graph.py]: feel free to keep a copy of the experimentation code as we can use it in our experimentation files

**Review by jessLryan — APPROVED** [review]

**Comment by jessLryan** [comment]
@GbotemiB can you please merge the single network function and make a new PR with the functions for multiple networks?

**Comment by GbotemiB** [comment]
> @GbotemiB can you please merge the single network function and make a new PR with the functions for multiple networks?

Hi @jessLryan, can you pleaase clarify

Do you mean i should merge the PR without the comparison for the two networks, then create another PR for the comparison for two networks

**Comment by jessLryan** [comment]
> > @GbotemiB can you please merge the single network function and make a new PR with the functions for multiple networks?
> 
> Hi @jessLryan, can you pleaase clarify
> 
> Do you mean i should merge the PR without the comparison for the two networks, then create another PR for the comparison for two networks

yes exactly :-)

**Comment by GbotemiB** [comment]
Hi @jessLryan, I have removed the recently added functions to compare two networks. But I cant merge it from my end. There is still a pending review from you.

**Review by jessLryan — APPROVED** [review]

**Pull request merged**
